python/cudf/cudf/core/frame.py

```diff
@@ -304,6 +304,44 @@ def _unaryop(self, op):
             result._data[name] = col.unary_operator(op)
         return result
 
+    def isnull(self):
+        """Identify missing values.
+        """
+        return self._is_null()
+
+    def isna(self):
+        """Identify missing values. Alias for `isnull`
+        """
+        return self._is_null()
+
+    def notnull(self):
+        """Identify missing values.
+        """
+        return self._is_valid()
+
+    def notna(self):
+        """Identify non-missing values. Alias for `notnull`.
+        """
+        return self._is_valid()
+
+    def _is_null(self):
+        result = self.copy()
```

Comment:
This is an unnecessary deep copy
```suggestion
        result = self.copy(deep=False)
```